java/src/apps/startup/TriggerRouteModelFactory.java

```diff
@@ -39,9 +41,14 @@ public void editModel(StartupModel model, Component parent) {
         if (this.getModelClass().isInstance(model)) {
             ArrayList<String> userNames = new ArrayList<>();
             InstanceManager.getDefault(RouteManager.class).getSystemNameList().stream().forEach((systemName) -> {
-                String userName = InstanceManager.getDefault(RouteManager.class).getBySystemName(systemName).getUserName();
-                if (userName != null && !userName.isEmpty()) {
-                    userNames.add(userName);
+                Route r = InstanceManager.getDefault(RouteManager.class).getBySystemName(systemName);
+                if (r != null) {
```

Comment:
`getDefault` is never supposed to return null.  That method means "I know there's a default there, or there's a problem".  `getOptionalDefault` can return null, as it means "If there's a default, please return it".

Not sure whether the right thing to do here is to use getOptionalDefault, or figure out why there isn't a RouteManager stored and fix that.  Generally, I think I'd prefer the latter, but it depends on what you're trying to do.